docs/LV_client/tools/mono_matlab_radar_plot.md

```diff
@@ -6,16 +6,22 @@
 
 ### Description
 
-Process Radar Data Cube and run a Matlab script to plot Radar data.
+Process radar data cube and run a Matlab script to plot Radar cube data.
 
 For technical support contact us at <b>support@monodrive.io</b> 
 
 ### Inputs
 
-- **Radar data cube:**  Data cube from Radar ouput 
+- **Radar Data Cube:**  Only send if, **send_cube** is set to true on the **Radar
```

Comment:
Should note somewhere that `"send_radar_cube"` must be true for this plotter to work

Reply:
yeah, good catch!